src/canopy_cli/cli.py

```diff
@@ -67,6 +65,14 @@ def check_server_health(url: str):
         )
         raise CLIError(msg)
 
+    except requests.exceptions.ReadTimeout:
+        msg = f"""
+        Canopy server did not send any data in the allotted
+        amount of time ({timeout_seconds} seconds).
+        Try restarting the server.
```

Comment:
Don't write "try restarting the server", that's not a viable solution.  
Please change it to something like "Please check that the server is running and that the provided url {url} is correct", or something of that sort.

The problem with the server getting "stuck" on Windows is a separate issue, which we should fix in its root cause.

Reply:
Done